Src/CSharpier.Cli/CommandLineFormatter.cs

```diff
@@ -206,25 +213,33 @@ async Task FormatFile(string actualFilePath, string originalFilePath)
                 }
 
                 var printerOptions = optionsProvider.GetPrinterOptionsFor(actualFilePath);
-                printerOptions.IncludeGenerated = commandLineOptions.IncludeGenerated;
 
-                await FormatPhysicalFile(
-                    actualFilePath,
-                    originalFilePath,
-                    fileSystem,
-                    logger,
-                    commandLineFormatterResult,
-                    writer,
-                    commandLineOptions,
-                    printerOptions,
-                    formattingCache,
-                    cancellationToken
-                );
+                if (printerOptions is not null)
+                {
+                    printerOptions.IncludeGenerated = commandLineOptions.IncludeGenerated;
```

Comment:
On an unrelated note, isn't it weird that everything else in CommandLineOptions is about control flow, and just this one is about formatting?

Should we maybe consider removing this option from CLI and have it only in the config files?

Reply:
I see it as similiar to directoryOrFile option `dotnet csharpier <directoryOrFile>` which is used to decide which files/folders to format. Currently all the options in in the config file are about what to do when a file is formatted, as opposed to what files to format. Although the new overrides option allows you to include extensions that aren't normally included.

`IncludeGenerated` does feel weird on `PrinterOptions`. I couldn't really find a better place to put it.

Long term I think csharpier should transition to what prettier does, where `<directoryOrFile>` uses globs. So the current command of `dotnet csharpier .` would be the equivalent of `dotnet csharpier **/*.{cs|cst}`